descriptions/css/properties/border-bottom-color.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "border-bottom-color": {
+        "__short_description": "The <strong><code>border-bottom-color</code></strong> CSS property sets the color of an element's bottom <a href='https://developer.mozilla.org/docs/Web/CSS/border'>border</a>. It can also be set with the shorthand CSS properties <a href='https://developer.mozilla.org/docs/Web/CSS/border-color'><code>border-color</code></a> or <a href='https://developer.mozilla.org/docs/Web/CSS/border-bottom'><code>border-bottom</code></a>."
```

Comment:
@wbamberg Given your comments on #23, should I drop the shorthand info here too? This is the reverse case: a non-shorthand property naming the shorthands that can set it.

Reply:
For short-descriptions, I think we should drop this, yes. I would be OK with leaving it in the Wiki page.

Reply:
Done. Thank you!